NgocMaiDang_democypress/cypress/e2e/addbook_spec.cy.js

```diff
@@ -0,0 +1,39 @@
+Cypress.on('uncaught:exception', (err, runnable) => {
+    // returning false here prevents Cypress from
+    // failing the test
+    return false
+});
+  
+describe('Add book with name "Git Pocket Guide" to my collection and verify it in my profile', () => {
+    it('Should add the book to the collection and verify it in the profile', () => {
```

Comment:
This case will not fail because you didn't deal with the browser alert, but it will not correct in the flow. When you run this case in the second time, the book is already in the collection, it will not prove that the flow add book are still working. Like I said before, you have to make sure that the book you want to add is not exist in collection, and the easiest way is delete all books from collection